src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -325,7 +325,7 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
                         } else {
                             ApplicationManager.getApplication().invokeLater {
                                 val snykCachedResults = getSnykCachedResults(project) ?: return@invokeLater
-                                val codeResultsLS = snykCachedResults.currentSnykCodeResultsLS ?: return@invokeLater
+                                val codeResultsLS = snykCachedResults.currentSnykCodeResultsLS
```

Comment:
We got rid of this null check because of the WIP [here](https://github.com/snyk/snyk-intellij-plugin/commit/84ab8626d25c005e5ac2e78b1171ef7cf0a2fd76#diff-ecd9359a648286ea597ac7a3648adb946b419e7021eb5249c80145c2c5a43cf9R52)?